history/titles/c_myehona.txt

```diff
@@ -1,7 +1,10 @@
-# c_assmel
-1.1.1 = {
-	holder=141200
+# e_storm_peaks
+1.1.1={
 }
+40.1.1={
+	liege=k_crystalsong_forest
+}
+40.1.1={holder=440002} #Thiassi[146000] # Lore character
```

Comment:
Can't you merge the same dates?

Reply:
I wouldn't do it because this line follows the format of the Character Generator... and probably I'm going to make something out of it soon.